frontend/public/components/utils/kebab.tsx

```diff
@@ -207,7 +207,9 @@ export type KebabOption = {
   href?: string, callback?: () => any;
   accessReview?: AccessReviewResourceAttributes;
 };
-export type KebabAction = (kind: K8sKind, obj: K8sResourceKind) => KebabOption;
+export type KebabAction = (kind: K8sKind, obj: K8sResourceKind, resources?: any) => KebabOption;
```

Comment:
Just curious, is `resources` expected to be an object? (Instead of an array?)

Reply:
it could be literary anything